src/main/java/com/conveyal/gtfs/validator/SpeedTripValidator.java

```diff
@@ -55,6 +55,8 @@ public void validateTrip(Trip trip, Route route, List<StopTime> stopTimes, List<
             Stop currStop = stops.get(i);
             // Distance is accumulated in case times are not provided for some StopTimes.
             distanceMeters += fastDistance(currStop.stop_lat, currStop.stop_lon, prevStop.stop_lat, prevStop.stop_lon);
+            // Redefine previous stop for next iteration (doing so here ensures the reassignment is not skipped if both stop times for the stop are missing).
```

Comment:
Line is too long; break comment into 2 lines.

Reply:
Updated.